sonar-project.properties

```
@@ -171,5 +172,6 @@ sonar.testExecutionReportPaths=packages/abap-deploy-config-inquirer/coverage/son
                                 packages/ui-service-sub-generator/coverage/sonar-report.xml, \
                                 packages/odata-service-inquirer/coverage/sonar-report.xml, \
                                 packages/jest-environment-ui5/coverage/sonar-report.xml, \
-                                packages/fiori-app-sub-generator/coverage/sonar-report.xml
+                                packages/fiori-app-sub-generator/coverage/sonar-report.xml, \
+                                packages/fiori-docs-embeddings/coverage/sonar-report.xml \
```

Invalid continuation character at the end
```suggestion
                                packages/fiori-docs-embeddings/coverage/sonar-report.xml
```